std/datetime.d

```diff
@@ -28337,9 +28337,15 @@ public:
                 setTZEnvVar("America/Los_Angeles");
                 assert(LocalTime().dstName == "PDT");
             }
-
             setTZEnvVar("America/New_York");
-            assert(LocalTime().dstName == "EDT");
+            version(NetBSD)
+            {
+                // The same bug on NetBSD 7+
+            }
+            else
+            {
+                assert(LocalTime().dstName == "EDT");
```

Comment:
I don't know why this would be failing on NetBSD and not FreeBSD (since presumably, the problem is code ported from one to the other, and "America/Los_Angeles" is broken on both), but I'd just move both this and the "America/New_York" into the previous version block, since `dstName` is clearly broken on both, and when the FreeBSD/NetBSD bug is fixed, we're going to have to verify that it's working on both before we can version these tests back in anyway.

Which reminds me, I have to figure out how on earth one goes about bugging the FreeBSD team about merging in a patch so that we can finally get this fixed on FreeBSD. The patch in the bug report has just sat there for something like 5 years even though it seems to fix the problem... :(